news/_posts/2019-09-11-prestashop-in-2019-and-beyond-part-2-pain-points.md

```diff
@@ -320,7 +321,7 @@ We have already started improving this in PrestaShop 1.7.6, but there is still s
 _Testing is doubting, by [CommitStrip](http://www.commitstrip.com/)_
 {: .text-center }
 
-This is one of the classic _consequences_ of complex, static, globally-stateful code. Since components are so tightly interweaved and depend on global state, putting the system in a state that allows to test a feature in an isolated and predictable way is really, really hard. In some cases, dependencies and hidden state can become extremely difficult and time-consuming to understand, specially when dealing with static in-memory cache (ie. variables that maintain their value between calls to a same method on a same thread).
+This is one of the classic _consequences_ of complex, static, globally-stateful code. Since components are so tightly interweaved and depend on global states, putting the system in a state that allows testing a feature in an isolated and predictable way is really, really hard. In some cases, dependencies and hidden state can become extremely difficult and time-consuming to understand, especially when dealing with static in-memory cache (i.e. variables that maintain their value between calls to the same method on the same thread).
```

Comment:
I cant see what changed 🤔 . Encoding ? ASCII letters ? Github shows nothing

Reply:
Ah, the `s`

Reply:
depend on global state -> depend on global states
a state that allows to test a feature ->a state that allows testing a feature
specially -> especially
calls to the same method on the same thread -> calls to a same method on a same thread

GitHub's diff sucks sometimes, try copy/paste in https://www.diffchecker.com/diff for instance :)

Reply:
Closing conversation for readability purpose :)